config/manager/manager.yaml

```diff
@@ -48,6 +48,8 @@ spec:
           value: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.10
         - name: GOVERNANCE_POLICY_FRAMEWORK_ADDON_IMAGE
           value: quay.io/stolostron/governance-policy-framework-addon:latest
+        - name: HOSTING_CLUSTER_NAME
+          value: cluster1
```

Comment:
What would set this value? Also, a hosted cluster can have different hosting clusters than other hosted clusters on the same hub.